src/client/src/orangehrmCorporateDirectoryPlugin/pages/CorporateDirectory.vue

```diff
@@ -0,0 +1,253 @@
+      <div class="orangehrm-paper-container">
+        <table-header :show-divider="false" :total="total"> </table-header>
+        <oxd-grid ref="scrollerRef" :cols="colSize" class="orangehrm-container">
+          <oxd-grid-item v-for="(employee, index) in employees" :key="employee">
+            <summary-card
+              :employee-designation="employee.employeeJobTitle"
+              :employee-id="employee.id"
+              :employee-location="employee.employeeLocation"
+              :employee-name="employee.employeeName"
+              :employee-sub-unit="employee.employeeSubUnit"
+              @click="showEmployeeDetails(index)"
+            >
+              <employee-details
+                v-if="isMobile && currentIndex === index"
+                :employee-id="employee.id"
+              >
+              </employee-details>
+            </summary-card>
+          </oxd-grid-item>
+          <oxd-loading-spinner
+            v-if="isLoading"
+            class="orangehrm-container-loader"
+          />
+        </oxd-grid>
+        <div class="orangehrm-bottom-container"></div>
+      </div>
+      <div
+        v-if="currentIndex > -1 && isMobile === false"
+        class="orangehrm-paper-container orangehrm-corporate-directory-sidebar"
+      >
+        <oxd-grid-item>
+          <summary-card-details
+            :employee-designation="employees[currentIndex].employeeJobTitle"
+            :employee-id="employees[currentIndex].id"
+            :employee-location="employees[currentIndex].employeeLocation"
+            :employee-name="employees[currentIndex].employeeName"
+            :employee-sub-unit="employees[currentIndex].employeeSubUnit"
+            @hideDetails="hideEmployeeDetails()"
+          ></summary-card-details>
+        </oxd-grid-item>
+      </div>
+    </div>
+  </div>
+</template>
+<script>
+import EmployeeAutocomplete from '@/core/components/inputs/EmployeeAutocomplete';
+import JobtitleDropdown from '@/orangehrmPimPlugin/components/JobtitleDropdown';
+import SubunitDropdown from '@/orangehrmPimPlugin/components/SubunitDropdown';
+import SummaryCard from '@/orangehrmCorporateDirectoryPlugin/components/SummaryCard';
+import SummaryCardDetails from '@/orangehrmCorporateDirectoryPlugin/components/SummaryCardDetails';
+import EmployeeDetails from '@/orangehrmCorporateDirectoryPlugin/components/EmployeeDetails';
+import {APIService} from '@/core/util/services/api.service';
+import Spinner from '@ohrm/oxd/core/components/Loader/Spinner';
+import useInfiniteScroll from '@ohrm/core/util/composable/useInfiniteScroll';
+import {reactive, toRefs} from 'vue';
+
+const employeeDataNormalizer = data => {
+  return data.map(item => {
+    return {
+      id: item.id,
+      employeeId: item.employeeId,
+      employeeName:
+        `${item.firstName} ${item.middleName} ${item.lastName}` +
+        (item.terminationId ? $t('general.past_employee') : ''),
```

Comment:
to fix lint issue;
you can't use $t outside from vue context. please refer https://github.com/orangehrm/orangehrm/blob/5.1/src/client/src/orangehrmPerformancePlugin/pages/PerformanceTrackers.vue#L129-L140

Reply:
done